assets/etc/oxide.conf

```diff
@@ -4,6 +4,9 @@ columns=6
 # Font size for application names
 fontSize=23
 
+# Required length for swipe activation
+swipeLength=30
+
```

Comment:
This setting should be stored in the system service's settings instead of here.
```suggestion
```

Reply:
fixed in latest